Cargo.toml

```diff
@@ -0,0 +1,37 @@
+[package]
+name = "rust-spiffe"
```

Comment:
The `spiffe` name is free on crates.io so I guess this could just be `"spiffe"`, when it will be used it will always be in a Rust context :)

Reply:
Good idea, it makes total sense!